app/geocode_api_parser.py

```diff
@@ -32,3 +32,6 @@ def get_place_info(self):
         self.formatted_address = geocode_info['results'][0]['formatted_address']
         self.place_type = geocode_info['results'][0]['address_components'][0]['types'][0]
         return {'place_type': self.place_type, 'formatted_address': self.formatted_address}
+
+    def check_api_is_available(self):
```

Comment:
You are supposed to be checking for the presence of the *API keys* not availability of the *API*.